doc/src/guides/solving/solve-numerically.md

```diff
@@ -0,0 +1,216 @@
+>>> from sympy.abc import x
+>>> nsolve(x**2 - 1, (-10, 0), solver='bisect')
+-1.00000000000000
+```
+
+## Solve a System of Equations Numerically
+
+### Solve Multidimensional Functions
+
+To solve multidimensional functions, supply a tuple of
+- functions `(f1, f2)`
+- variables to solve for `(x1, x2)`
+- starting values `(-1, 1)`
+
+```py
+>>> from sympy import Symbol, nsolve
+>>> x1 = Symbol('x1')
+>>> x2 = Symbol('x2')
+>>> f1 = 3 * x1**2 - 2 * x2**2 - 1
+>>> f2 = x1**2 - 2 * x1 + x2**2 + 2 * x2 - 8
+>>> print(nsolve((f1, f2), (x1, x2), (-1, 1)))
+Matrix([[-1.19287309935246], [1.27844411169911]])
+```
+
+### Increase Precision of the Solution
+
+You can increase the precision of the solution using `prec`:
+
+```py
+>>> from sympy import Symbol, nsolve
+>>> x1 = Symbol('x1')
+>>> x2 = Symbol('x2')
+>>> f1 = 3 * x1**2 - 2 * x2**2 - 1
+>>> f2 = x1**2 - 2 * x1 + x2**2 + 2 * x2 - 8
+>>> print(nsolve((f1, f2), (x1, x2), (-1, 1), prec=25))
+Matrix([[-1.192873099352460791205211], [1.278444111699106966687122]])
+```
+
+### Create a Function That Can Be Solved With SciPy
+
+As noted above, SymPy focuses on symbolic computation and is not optimized for
+numerical calculations. A recommend use case for SymPy in a workflow is as given
+below:
+- use SymPy to generate (by symbolically simplifying or solving an equation) the
+  mathematical expression
+- use a numerical library such as SciPy to generate numerical solutions
+
+```py
+>>> from sympy import simplify, cos, sin
+>>> from sympy.abc import x, y
+>>> expr = cos(x * (x + x**2)/(x*sin(y)**2 + x*cos(y)**2 + x))
+>>> simplify(expr)
+cos(x*(x + 1)/2)
+>>> from sympy.utilities.lambdify import implemented_function
+>>> from sympy import lambdify
+>>> f = implemented_function('f', lambda x: cos(x*(x + 1)/2))
+>>> lam_f = lambdify(x, f(x))
+>>> from scipy import optimize
+>>> sol = optimize.root_scalar(lam_f, bracket=[0, 2], method='brentq')
+>>> sol.root
+1.3416277185114782
+```
+
+## Use the Solution Result
+
+### Substitute the Result Into an Expression
+
+The best practice is to use {func}`~sympy.core.evalf` to to substitute numerical
+values into expressions. Using [`subs`](sympy.core.basic.Basic.subs) can give
+incorrect result due to precision errors:
+
+```py
+>>> from sympy import cos, nsolve, Symbol
+>>> x = Symbol('x')
+>>> f = cos(x) - x
+>>> x_value = nsolve(f, x, 1); x_value
+0.739085133215161
+>>> y = Symbol('y')
+>>> z = Symbol('z')
+>>> values = {x: x_value, y: 1e16, z: 1e16}
+>>> (x + y - z).evalf(subs=values) # Gives correct result
+0.739085133215161
+>>> (x + y - z).subs(values) # Gives incorrect result due to precision errors
+0
+```
+
+You can also leave some symbols as variables:
+
+```py
+>>> from sympy import cos, nsolve, Symbol
+>>> x = Symbol('x')
+>>> f = cos(x) - x
+>>> x_value = nsolve(f, x, 1); x_value
+0.739085133215161
+>>> y = Symbol('y')
+>>> z = Symbol('z')
+>>> g = x * y**2
+>>> values = {x: x_value, y: 1}
+>>> (x + y - z).evalf(subs=values)
```

Comment:
I thought this was broken https://github.com/sympy/sympy/issues/6974. Maybe that issue is fixed now?